### PR TITLE
Sca 1877 move account to common tactical yaml

### PR DESCRIPTION
**Pull request by davidhughes1-ccs**
Tidying up the 'account' calls which should be both tactical and common.

### CODE REVIEW

File: bat/oauth.yaml [hunk 1]
```diff
@@ -57,7 +57,152 @@ paths:
...
+            application/json:
+              schema:
+                $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-scale-api-definitions/master/common/CCS_Standard_API_Schema.yaml#/components/schemas/Errors'
+      security:
+        - bearerAuth: []
+    post:
+      description: >-
+        Creates a new account
+
+        This endpoint requires [Spree Auth Devise](https://github.com/spree/spree_auth_devise) gem installed
```
Comment (adrianmilne) [inline bat/oauth.yaml:98]: Minor point - suggest maybe this implementation detail is not relevant in the context of our SCALE layer API

File: bat/oauth.yaml [hunk 2]
```diff
@@ -57,7 +57,152 @@ paths:
...
+          content:
+            application/json:
+              schema:
+                $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-scale-api-definitions/master/common/CCS_Standard_API_Schema.yaml#/components/schemas/Errors'
+
+    patch:
+      description: >-
+        Updates user account
+
+        This endpoint requires [Spree Auth Devise](https://github.com/spree/spree_auth_devise) gem installed
```
Comment (adrianmilne) [inline bat/oauth.yaml:146]: (see note above)

File: bat/oauth.yaml (hunk 1 above)
Comment (davidhughes1-ccs) [reply bat/oauth.yaml:98]: Yes, true.
Will remove

**Review by adrianmilne — APPROVED** [review]
👍 Great stuff - thanks for splitting! Just that one minor observation re Spree gems

**Review by tommyb82 — APPROVED** [review]
👍

**Pull request merged**